app/pods/publications/paused/route.js

```diff
@@ -0,0 +1,8 @@
+import Route from '@ember/routing/route';
+
+export default class PausedRoute extends Route {
+  model() {
+    // Normally we would query store here, but for now, we get the mocks
```

Comment:
Dit is een copy paste, maar nergens wordt nog data gemocked dus eerder verwarrend.
In deze route gaan we volgens mij nooit data inladen, omdat we dat doen in de 2 subroutes via-MR en niet-via-MR.

Reply:
Yes, ik ga de comment wegdoen maar de file nog laten. We kuisen dit binnenkort in zijn geheel op met de nieuwe designs.